test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -405,6 +405,85 @@ TEST_VM(os, jio_snprintf) {
   test_snprintf(jio_snprintf, false);
 }
 
+#ifndef MAX_PATH
+#define MAX_PATH    (2 * K)
+#endif
+
+TEST_VM(os, realpath) {
+  /* POSIX requires that the file exists, Windows doesn't */
```

Comment:
I'm not sure what this comment means. I can't fully discern from

https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/fullpath-wfullpath?view=msvc-170

what Windows does if the path does not exist.

Reply:
I can only report by example - on my local machine, Windows doesn't appear to care if the file exists or not, but according to the docs you point to, Windows **does** care if the drive letter is valid.
Adding an invalid drive letter test is beyond the current scope of this test.
I have expanded this comment to be more explanatory:

`// POSIX requires that the file exists; `
`// Windows tests for a valid drive letter but may or may not test if the file exists.`